### PR TITLE
Update body-english

### PR DESCRIPTION
**Pull request by ericanaughton**
:cactus: 

ready to go

### CODE REVIEW

File: elements/body.html [hunk 1]
```diff
@@ -1,20 +1,14 @@
 <article class="article">
   <header class="article-header">
     <h2>
-      <code>
-        <!-- Actual Tag Name  -->
-        &lt;body>
-      </code>
-      <!-- Audio -->
+      <code>&lt;body&gt;</code>
       <audio src="audio/body.mp3" controls></audio>
```
Comment (nwerbeckes-gpsw) [inline elements/body.html:7]: I noticed you removed audio from several other elements.  What's the reason for leaving it in for this one?

Comment (ericanaughton) [inline elements/body.html:7]: body, header, footer, a, and br all have audio recordings (those are the 5 elements that the yweb group completed). @nwerbeckes-gpsw

**Comment by nwerbeckes-gpsw** [comment]
:+1:

**Comment by jremsik-gpsw** [comment]
:+1:

**Pull request merged**